staging/src/kubevirt.io/client-go/examples/listvms/README.md

```diff
@@ -0,0 +1,23 @@
+# listvms
+
+List all VMs and VMIs objects in KubeVirt
+
+## How to build
+```
+$ export GO111MODULE=on
+$ go mod init
+$ go get k8s.io/apimachinery@kubernetes-1.14.0
```

Comment:
So we have this dependency, but it is not pulled in already? Why does this have to be fetched extra?

Reply:
I tried to build this example in separate folder, maybe due to old cached code, I was getting some error, which requires to `apimachinery@kubernetes-1.14.0` to fixed.

Again, since these examples will be tried by someone who may not have KubeVirt code, so I included this step.

Reply:
Hm, if client-go has a dependency, then it should be included in the client go go.mod file already. Maybe we miss it there. Can be that it is in go.mod of our root folder and that it can therefore be compiled, but that would be wrong (we also need it then in go.mod in the staging folder of client-go).

Reply:
I removed the line. Earlier, I mentioned all steps required by just `listvms.go` file. Since `go.mod` file is there, we don't need these lines.

Reply:
Ok, I am still not sure if this is resolved, since you had to do the `go get`. You should not have had to do that. `go mod init` should have pulled it in when you ran it the first time. If that did not pull in all dependencies we need to fix that.

Reply:
I'll give it a try

Reply:
Yes, there was problem. I fixed it by adding line 
```
 replace k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
````
from `go.mod` of kubevirt to listvms `go.mod`. Now this it fetches the dependencies correctly & builds the example without any extra go get.

Reply:
Ok, so this means that we miss this dependency in `go.mod` of `client-go`.

Reply:
It is there in the `client-go` `go.mod` file too https://github.com/kubevirt/kubevirt/blob/master/staging/src/kubevirt.io/client-go/go.mod#L61, I am not sure, while building example, why it fetches old version.

Reply:
hm, ok, we should resolve this somehow. For now works for me how it is.